packages/css/src/components/switch/switch.scss

```diff
@@ -51,10 +53,11 @@
 }
 
 .ams-switch__input:checked + .ams-switch__label::before {
-  // The 2px is to account for the 1px padding-inline on the label
-  transform: translate(calc(var(--ams-switch-width) - var(--ams-switch-thumb-width) - 2px));
+  transform: translate(
+    calc(var(--ams-switch-width) - var(--ams-switch-thumb-width) - 2 * var(--ams-switch-track-padding))
+  );
 }
 
 .ams-switch:hover .ams-switch__input:enabled + .ams-switch__label::before {
-  box-shadow: 0 0 0 2px var(--ams-switch-thumb-hover-color);
+  box-shadow: 0 0 0 var(--ams-switch-thumb-hover-box-shadow-spread-radius) var(--ams-switch-thumb-hover-color);
```

Comment:
Do we want to turn this into a single token? We can use `ams-switch-thumb-hover-color` there as well.

Reply:
I don’t understand, please elaborate.

Reply:
We could set something like `box-shadow: var(--ams-switch-thumb-hover-box-shadow);` here.

The token could be `0 0 0 0.125rem {ams.switch.thumb.hover.color}`.

That results in somewhat less concatenation.

(Pseudo code)

Reply:
Ah yes, much better.